docs/source/posts/2021/2021-07-19-week-7-sajag.rst

```diff
@@ -0,0 +1,62 @@
+Sixth week of coding!
+=====================
+
+.. post:: July 19 2021
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Welcome to the seventh weekly check-in. I'll be sharing my progress for the sixth week of coding.
+
+What did you do this week?
+--------------------------
+
+#. Updated `Molecular module`_: made it more pythonic, implemented
+   ribbon actor, added support to pass numpy arrays (earlier, atomic
+   data could only be added by using the add_atom)
+#. Created a `PR`_ to -
+
+   -  update the helical motion animation to use a single line actor,
+      added textblocks to display velocity of the particle. Preview -
+      |image1|
+   -  For brownian motion animation, I removed rotation(azimuth) and box
+      actor, added textblock to display the number of particles and to
+      show the simulation steps. Preview -
+
+      |image2|
+
+#. Updated surface animation (used gridUI, added multiple animations).
+   Preview -
+   |image3|
+#. Created a `topic`_ on vtk discourse forum to query about gaps in
+   bonds (tried resolving it by manipulating vtkProperties:
+   BackfaceCulling, FrontfaceCulling but was unsuccessful)
+#. Read about molecular surface (theory behind it)
+
```

Comment:
really badly rendered. Need to review the whole block

Reply:
reviewed and changed